crd/apis/network/v1/network.go

```diff
@@ -18,6 +18,13 @@ func IsDefaultNetwork(networkName string) bool {
 	return networkName == DefaultNetworkName || networkName == DefaultPodNetworkName
 }
 
+// IsDefaultNetworkOrEmpty takes a network name and returns if it is a default network or empty.
+// Both DefaultNetworkName and DefaultPodNetworkName are considered as default network for compatibility purposes.
+// DefaultNetworkName will eventually be removed.
+func IsDefaultNetworkOrEmpty(networkName string) bool {
+	return networkName == "" || networkName == DefaultNetworkName || networkName == DefaultPodNetworkName
```

Comment:
Could we just: `networkName= "" | IsDefaultNetwork(networkName)`? this way when DefaultNetworkName gets depricated, we will have to change just in 1 place.

Reply:
Done, thank you!